django/website/logframe/migrations/0004_auto_20151215_1203.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_logframe_slug(apps, schema_editor):
+    LogFrame = apps.get_model('logframe', 'LogFrame')
+    for logframe in LogFrame.objects.all():
+        slug = logframe.name.lower()
+        slug.replace(' ', '_')
+        logframe.slug = slug
+        logframe.save()
```

Comment:
1. SlugField only allows "letters, numbers, underscores or hyphens" (see https://docs.djangoproject.com/en/1.8/ref/models/fields/#slugfield ). The code here could potentially create other values. Also SlugField is limited to 50 chars by default;

2. Also the field is 'slug' on LogFrame model is marked as unique. The code here doesn't guarantee that (I realise LogFrame.name is unique, however since we manipulate it we loose it's uniqueness).